backend/routers/admin_analytics.py

```
@@ -13,38 +16,71 @@
+            "sessions": await db.fetchval("SELECT COUNT(*) FROM sessions"),
+        }
+    except HTTPException as e:
+        raise e
+    except Exception as e:
+        logger.exception(f"Error in /analytics endpoint: {e}")
+        raise HTTPException(status_code=500, detail="Internal Server Error") from e
 
 @router.get("/revenue-insights")
 async def revenue_insights(request: Request, db=Depends(get_db)):
-    admin_user = AuthenticationHelper.verify_admin_access_strict(request)
+    admin_user = await AuthenticationHelper.verify_admin_access_strict(request, db)
```

Ruff (0.12.2)

Local variable `admin_user` is assigned to but never used

Remove assignment to unused variable `admin_user`

(F841)